data_steward/spec/pages/file_transfer_procedures.md

```diff
@@ -1,43 +1,98 @@
+
+__Step 1:__ Log out of any currently active Google IDs. Log into MyAccount (https://myaccount.google.com/)  with the pmi-ops.org credentials provided by the DRC User Admin.
+
+__Step 2:__ Data Stewards will receive an email from the DRC User Admin containing a direct link to the assigned Bucket. Open a web browser and navigate to the Google Cloud Storage Bucket via the direct link shared with you by the DRC User Admin. 
+
+NOTE: If you receive the error message shown below, please check that the pmi-ops.org account is the active account. 
+
+__PICTURE FROM SOP NEEDS INSERTED HERE__
+
+__Step 3:__ Create a folder within the Bucket. Data files for each upload cycle must be contained within a unique folder. Loose files uploaded to the bucket root will not be accepted. 
+
+__PICTURE FROM SOP NEEDS INSERTED HERE__
+
+__Step 4:__ Uploading with gsutil
+- Authenticate:Enter gcloud auth login
+  - This opens a browser window to authenticate to Google Cloud Platform. Login using the assigned pmi-ops account (e.g. john.smith@pmi-ops.org)
+ - Set project: gcloud config set project all-of-us-staging (for example)
+ - Upload files: gsutil cp -r my-site-ehr-data-2017-05-20 gs://aou-ehr-test-awardee-site-123/
+ - Upload
+
+__PICTURE FROM SOP NEEDS INSERTED HERE__
+
+NOTE: for instructions on installing gsutil [click here](https://cloud.google.com/storage/docs/gsutil_install#sdk-install).
+
+_The following is an alternative Upload Procedure to using gsutil_
+ - In a desktop environment, navigate to the data set. Drag and drop the data set files from the local folder into the browser window where the Google Cloud Storage Browser is open to the folder created in Step 3 under Procedures.
+
+__Step 5:__ Within six (6) hours of uploading data to a folder within the Bucket, the [report page](https://storage.googleapis.com/test-aou-curation-spec-site-bucket/report.html) on the All of Us Upload website will display the sites uploaded tables with feedback on whether it was received or loaded. Review this status to verify that the upload was successful.  This website is also provided by the sysadmin in the setup of the pmi-ops account.
+
+__Step 6:__ If any errors were detected in the data files uploaded to the Bucket, a detailed report will appear in the Bucket folder to which the files were uploaded. Navigate to the Bucket folder containing the recently uploaded data files and review the report. Resolve in the data files any errors identified and repeat steps 1-5 to reupload.
+
+---
+
+## ISSUE REPORTING
+Report any issues with Bucket access or the upload process to sysadmin@pmi-ops.org. A member of the DRC team will contact the Data Steward to assist with resolution of the issue.
```

Comment:
Do we want all issues sent to an email? Is it possible to use Jira?

Reply:
Well it won't be Columbia, it will be the DRC and our security team until the volume gets out of hand.

Reply:
@cmcphilamy do you know what the plan is when the volume does get out of hand? I think it might be a good idea to put that in the SOP before the code freeze since it will be difficult to make system changes afterwards. My apologies if there are other considerations I am unaware of.

Reply:
@mark-velez I will bring it up with the group.  The main concern is that sites would identify themselves using JIRA because JIRA is viewable by a large number of people.